velox/type/Filter.h

```diff
@@ -759,6 +830,13 @@ class HugeintRange final : public Filter {
         nullAllowed_ ? "with nulls" : "no nulls");
   }
 
+  bool testingEquals(const Filter& other) const final {
+    auto ptr = dynamic_cast<const HugeintRange*>(&other);
+    return Filter::testingEquals(other) && ptr != nullptr &&
+        (std::to_string(lower_) == std::to_string(ptr->lower_)) &&
```

Comment:
Why do we have to convert lower_ and upper_ to strings to check for equality?

Reply:
Fixed